tms_objects2.erb

```diff
@@ -334,6 +334,17 @@ xsi:schemaLocation="http://www.lido-schema.org http://www.lido-schema.org/schema
               <lido:term><%= o["Period"] %></lido:term>
             </lido:periodName> 
         <% end %>
+            <lido:eventMaterialsTech> 
+              <lido:displayMaterial><%= MediumPS %></lido:displayMaterial> 
+              <lido:materialsTech> 
+        <% AATMediumTerms.each do |m| %> 
+                <lido:termMaterialTech> 
+                  <lido:conceptID lido:type = "<%= m["colThesxrefType"] %>" lido:source = "<%= m["UCTS"] %>"><%= m["FinalST"] %></lido:conceptID> 
```

Comment:
just for consistency remove space between attributes and equal sign, should be like: lido:type="...